### PR TITLE
Support -d|--domain optional parameter in the shell bundle

### PR DESCRIPTION
**Pull request by robbiezhang**
Support -d in omsadmin.sh for onboarding and heartbeat
Customer can specify the top level domain in the parameter
to onboard the omsagent agent in FairFax, MoonCake, BlackForest

### CODE REVIEW

**Comment by robbiezhang** [comment]
@Microsoft/omsagent-devs @varunkumta

File: installer/bundle/bundle_skel.sh [hunk 1]
```diff
@@ -573,13 +572,6 @@ if [ "$ONBOARD_ERROR" -ne 0 ]; then
     exit 1
 fi
 
-if [ "$onboardINT" -ne 0 ]; then
-    if [ -z "$onboardID" -o -z "$onboardKey" ]; then
```
Comment (varunkumta) [inline installer/bundle/bundle_skel.sh]: add this check below, since if -d was specified it means they want to onboard and must specify the id and key

**Comment by varunkumta** [comment]
Signed off

File: installer/bundle/bundle_skel.sh [hunk 2]
```diff
@@ -60,6 +60,7 @@ usage()
     echo
     echo "  -w id, --id id         Use workspace ID <id> for automatic onboarding."
     echo "  -s key, --shared key   Use <key> as the shared key for automatic onboarding."
+    echo "  -d dmn, --domain dmn   Use <dmn> as the domain for onboarding. Optional."
```
Comment (agup006) [inline installer/bundle/bundle_skel.sh]: Can we specify it is an Azure domain

File: installer/scripts/omsadmin.sh [hunk 3]
```diff
@@ -68,7 +68,7 @@ usage()
     echo
     echo "Maintenance tool for OMS:"
     echo "Onboarding:"
-    echo "$basename -w <workspace id> -s <shared key>"
+    echo "$basename -w <workspace id> -s <shared key> -d <top level domain>"
```
Comment (niroyb) [inline installer/scripts/omsadmin.sh]: If the -d flag is optional it should be in square brackets like in man pages

**Comment by niroyb** [comment]
Signed off

File: installer/bundle/bundle_skel.sh (hunk 2 above)
Comment (robbiezhang) [inline installer/bundle/bundle_skel.sh]: It should be OMS domain, e.g. opinsights.azure.com, opinsights.azure.us, etc...

**Pull request merged**